Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:latest as unwindtest
+
+# Check the working directory
+RUN ls -altr
+#RUN addgroup -S appuser
```

Comment:
consider remove the comments

Reply:
sure .. final changes . I will remove these

Reply:
All changes are removed